now-usability.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 2,
+    "public": true,
+    "scope": "aragon",
+    "alias": "court-rinkeby.aragon.org",
```

Comment:
Should we call this `court-usability.aragon.org` to avoid confusing it with the actual Court Rinkeby instance?

Reply:
So far we've been pretending usability is the rinkeby instance, and that we'd do a switchover later at some point.

Perhaps it would be better as `court-mock.aragon.org` or `mock.court.aragon.org` instead though.

Reply:
I was suggesting the other cause we followed that terminology for all the other pieces of each env already 😅 

I prefer leaving it as `rinkeby` if you prefer not using `usability` then